src/main/java/com/Acrobot/ChestShop/Events/SignValidationEvent.java

```diff
@@ -0,0 +1,65 @@
+package com.Acrobot.ChestShop.Events;
+
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+/**
+ * Represents a request whether a name is allowed.
+ */
+public class SignValidationEvent extends Event {
+    private static final HandlerList handlers = new HandlerList();
+    private final String[] lines;
+    private final String ownerName;
```

Comment:
Why is this separate? It is already contained in the `lines`, I don't think there should be a duplication of the information.

Reply:
Yes, either remove it or add utility methods for each of the 4 lines to query the value from the array.

But storing the value two times should not be done.